### PR TITLE
Fix #2582 - Backport ability to log user and date to 1.6

### LINKED ISSUE #2582: Backport Ability to log the last update to a feature (#2156) to GMF 1.6
### Functionality descripton

During editing, the user and date should be saved together with the changes into the database, in the same table as the geodata. 
- The logs are filled when a new feature is created and when a feature is changed
- The field date is of type Timestamp
- The field user is of type String
- be able to configure if a table is logged or not.
- if a table is logged, it can be assumed that the fields “date” and “user” are present.
- The customer will create the fields “date” and “user” in the geotables.

Reasons for this option (and why we do not do a generic Metadata table): If the table has a constraint of NOT NULL values, which is filled by a trigger from the log table, it is not possible to fill from this second transaction, because the first state will be incoherent.
### To Do

The developements have already be done in GMF 2.0 by @fredj. As the customer cannot install GMF 2.x due to infrastruction restrictions, he needs this feature in GMF 1.6.x. Therefore you need to backport the PR #2156 to the 1.6 branch.

The feature is installed / testable in GMF 2.x here: 
- Login admin/c2c
- Layer "Polygon": https://gmf-test.sig.cloud.camptocamp.net/baselstad/edit/?map_x=542000&map_y=154000&map_zoom=3&tree_group_layers_Edit=polygon&tree_groups=Edit
- Configuration in the admin interface (UI Metadata): https://gmf-test.sig.cloud.camptocamp.net/baselstad/wsgi/admin/LayerV1/71/edit

Developement server: gmf-test.sig.cloud.camptocamp.net

### PR DESCRIPTION
**Pull request by juliensam**

### CODE REVIEW

File: c2cgeoportal/views/layers.py [hunk 1]
```diff
@@ -362,9 +388,21 @@ def metadata(self):
...
+        if last_update_date:
+            exclude.append(last_update_date)
+        last_update_user = self._get_ui_metadata(layer, "lastUpdateUserColumn")
+        if last_update_user:
+            exclude.append(last_update_user)
+
         return get_class(
             str(layer.geo_table),
-            exclude_properties=layer.exclude_properties
+            exclude_properties=','.join(exclude)
```
Comment (fredj) [inline c2cgeoportal/views/layers.py]: please use double quotes: `exclude_properties=",".join(exclude)` (travis error)

**Comment by juliensam** [comment]
I'm not sure what coveralls test. Which tests (6?) are failing?

**Comment by fredj** [comment]
@sbrunner can we ignore the coveralls test? I yes, I think it's good to merge.

**Comment by juliensam** [comment]
I'll wait for @sbrunner feedback before merging.

**Comment by sbrunner** [comment]
@juliensam can you fix the author on the two first commits?

this https://github.com/camptocamp/c2cgeoportal/commit/4b6199b7e4718e3c0f6c14d9ca1c0f5d74d3ea43 should also be backported :-)

Finally can you merge all the commits in one commit?

**Comment by juliensam** [comment]
Commit squashed and the layer tests now pass. It seems that Travis is raising a few errors in other parts of the code. I suggest to fix those in another pull request.

In short, ready to review.

**Comment by sbrunner** [comment]
#2640 should fix the Travis failure, can you do rebase?

**Comment by juliensam** [comment]
Ready to review

**Comment by sbrunner** [comment]
Thanks :-)

**Pull request merged**